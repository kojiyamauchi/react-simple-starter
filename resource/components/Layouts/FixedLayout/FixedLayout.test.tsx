/**
 * @jest-environment jsdom
 */
/*
  Fixed Layout Component Unit Test.
*/

import { FixedLayout } from './'
import { BrowserRouter } from 'react-router-dom'
import { render /* ,screen */ } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
// import userEvent from '@testing-library/user-event'

describe('Fixed Layout Component Unit Test', () => {
  it('Snapshot Testing', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <FixedLayout pages="ui-test" />
      </BrowserRouter>
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
