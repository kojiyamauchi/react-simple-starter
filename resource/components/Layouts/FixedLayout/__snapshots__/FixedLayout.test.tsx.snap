// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fixed Layout Component Unit Test Snapshot Testing 1`] = `
<DocumentFragment>
  <div
    class="sc-eCApnc gQSlmJ ui-test layout fixed-layout"
  >
    <header
      class="sc-dlnjwi eseKde"
    >
      <h1
        class="sc-bdnxRM kSZPnN primary-heading"
      >
        React Simple Starter
      </h1>
      <ul
        class="sc-gtsrHT bQjmdZ"
      >
        <li>
          <a
            class="link-letter"
            href="/react-simple-starter/"
          >
            Primary Page
          </a>
        </li>
        <li>
          <a
            class="link-letter"
            href="/react-simple-starter/second"
          >
            Secondary Page
          </a>
        </li>
        <li>
          <a
            class="link-letter"
            href="/react-simple-starter/third"
          >
            Third Page
          </a>
        </li>
        <li>
          <a
            class="link-letter"
            href="/react-simple-starter/fourth"
          >
            Fourth Page
          </a>
        </li>
      </ul>
    </header>
    <main
      class="main-content"
    />
    <footer
      class="sc-hKFxyN cPdexJ"
    >
      <a
        class="link-letter"
        href="http://kojiyamauchi.com"
        rel="noreferrer"
        target="_blank"
      >
        Â© 2021 All The Small Things, All Rights Reserved.
      </a>
    </footer>
  </div>
</DocumentFragment>
`;
